mesonbuild/utils/universal.py

```diff
@@ -1902,7 +1902,7 @@ def _make_tree_writable(topdir: T.Union[str, Path]) -> None:
         os.chmod(d, os.stat(d).st_mode | stat.S_IWRITE | stat.S_IREAD)
         for fname in files:
             fpath = os.path.join(d, fname)
-            if os.path.isfile(fpath):
+            if not os.path.islink(fpath) and os.path.isfile(fpath):
```

Comment:
Why don't you simply add `follow_symlinks=False` to `chmod` ?

Reply:
Like I said, symlinks don't have permissions on posix, trying to change them is just weird so we shouldn't try to do it in the first place.